packages/cxl-ui/scss/cxl-course-card.scss

```diff
@@ -0,0 +1,197 @@
+:host {
+  display: block;
+  padding: var(--lumo-space-m) var(--lumo-space-l);
+  background: var(--lumo-tint);
+  border: 1px solid var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius-s);
+  box-shadow: var(--lumo-box-shadow-xs);
+  break-inside: avoid;
+  transform: translateZ(0); // CSS columns @see https://stackoverflow.com/a/55110789/35946
+
+  .container {
+    display: none;
+  }
+}
+
+:host(:hover) {
+  border-color: var(--lumo-primary-color);
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host(:first-child) {
+  margin-top: unset;
+}
+
+:host(:last-child) {
+  margin-bottom: unset;
+}
+
+:host([theme~="dark"]) {
+  background-color: var(--lumo-contrast);
+}
+
+/**
+ * CXL 2.0
+*/
+
+:host([theme~="cxl-course"]) {
```

Comment:
I think these should be default styles for a card which is named `cxl-course-card`. At the same time we need other color variations and for them I suggest to create themes:
* video,
* playbook,
* training.

Reply:
I completely missed the different background in figma. My bad. However, we already have to set `type="Course|Video|Training|Playbook"`. That attribute is used both to determine the type in logic and as a text value for the first tag at the top of the header. Can I just use that to set the CSS background?